evidence/doc.go

```diff
@@ -0,0 +1,37 @@
+/*
+Package evidence handles all evidence storage and gossiping from detection to block proposal.
+For the different types of evidence refer to the `evidence.go` file in the types package
+or https://github.com/tendermint/spec/blob/master/spec/consensus/light-client/accountability.md.
+
+## Gossiping
+
+The core functionality begins with the evidence reactor (see reactor.
+go) which operates both the sending and receiving of evidence.
+
+The `Receive` function takes a list of evidence and does the following:
+ 1. breaks it down into individual evidence if it is `Composite Evidence` (see types/evidence.
+go#ConflictingHeadersEvidence)
+ 2. checks that it does not already have the evidence stored
+ 3. verifies the evidence against the nodes state (see state/validation.go#VerifyEvidence)
+ 4. stores the evidence to a db and a concurrent list
+
+The gossiping of evidence is initiated when a peer is added which starts a go routine to broadcast currently
+uncommitted evidence at intervals of 60 seconds (set by the by broadcastEvidenceIntervalS).
+It uses a concurrent list to store the evidence and before sending verifies that each evidence is still valid in the
+sense that it has not exceeded the max evidence age and height. This should be set to be equal to the "trusting
```

Comment:
not trusting period, but unbonding period.. anyway, I don't think this is the right place for it.

Reply:
Can we try be consistent with the terminology - I though it was unbonding period for the SDK and trust period for Tendermint. Or if you think the concept of trust period shouldn't exist in this context then I can remove it

Reply:
> I though it was unbonding period for the SDK and trust period for Tendermint.

sorry. but these are not the same.

trusting period = 2/3 * unbonding period

Reply:
> Or if you think the concept of trust period shouldn't exist in this context then I can remove it

please do